config/routes.rb

```diff
@@ -16,6 +16,8 @@
   get 'tasks', to: 'tasks#my'
   get 'tasks/organisation', to: 'tasks#organisation'
 
+  get 'api/legacy_sync', to: 'sync#legacy'
+
```

Comment:
It's only temporary, so I won't be too pedantic about it, but it's often a good idea to name API calls with a verb if it's an action with changes (and make it a POST too), eg 'start_sync' or something.

However could you add a comment about what this does and the fact it's temporary?

Reply:
Yes of course - will change this